Added 32B stage2 checkpoints .csv (#220)
allenai/OLMo-core: README.md

You can override most configuration options from the command-line. For example, to override the learning rate you could launch the script like this:

```bash

> torchrun --nproc-per-node=8 ./src/scripts/train/OLMo-2-0325-32B-train.py run01

Reviewer: ```suggestion
torchrun --nproc-per-node=8 ./src/scripts/train/OLMo-2-0325-32B-train.py run01 --train_module.optim.lr=6e-3
```

```

> To continue annealing from a checkpoint, launch the script like this:

Reviewer: ```suggestion
To continue annealing from a checkpoint, we use a separate script which can be launched like this:
```

| Variant         | OLMo Format (Stage 1)                                                                                         | OLMo Format (Stage 2) | Hugging Face Format                                                               |
|----------------|-----------------------------------------------------------------------------------------------------|--------|----------------------------------------------------------------------------------|

> | **OLMo-2 32B**  | [OLMo-2 32B](https://github.com/allenai/OLMo-core/blob/main/src/scripts/official/OLMo-2-0325-32B.csv)     | [OLMo-2 32B](./src/scripts/official/OLMo-2-0325-32B-stage2.csv)      | [Hugging Face for the 32B variant](https://huggingface.co/allenai/OLMo-2-0325-32B)  |

Reviewer: ```suggestion
| **OLMo-2 32B**  | [OLMo-2 32B](https://github.com/allenai/OLMo-core/blob/main/src/scripts/official/OLMo-2-0325-32B.csv)     | [OLMo-2 32B](https://github.com/allenai/OLMo-core/blob/main/src/scripts/official/OLMo-2-0325-32B-stage2.csv)      | [Hugging Face for the 32B variant](https://huggingface.co/allenai/OLMo-2-0325-32B)  |
```

Reviewer: Should be a URL so the link works on PyPI
